acceptance/features/migration.feature

```diff
@@ -0,0 +1,25 @@
+Feature: operator can migrate/adopt Redpanda helm base deployment
+
+  @skip:gke @skip:aks @skip:eks
+  Scenario: Adopt or migrate from stand alone helm chart release to Redpanda custom resource
+        Given there is "test-migrate-from-helm" Redpanda helm release with values:
+        """
+        fullnameOverride: name-override
+        """
+
+        When I apply Kubernetes manifest:
+        """
+    # tag::redpanda-custom-resource-manifest[]
+        # This manifest is a copy of Redpanda release helm values
```

Comment:
I think this comment will be much more useful if we move it to be directly below `clusterSpec` as it's just the clusterSpec that's a direct copy of the helm values.

Reply:
ok